middle_end/flambda/from_lambda/cps_conversion.ml

```diff
@@ -1050,30 +1051,31 @@ and cps_switch (switch : L.lambda_switch) ~scrutinee (k : Continuation.t)
         }
       in
       let is_scrutinee_int = Ident.create_local "is_scrutinee_int" in
-      let isint_switch = I.Let ( is_scrutinee_int,
-      Not_user_visible,
-      Pgenval,
-      Prim { prim = Pflambda_isint;
-        args = [Var scrutinee];
-        loc = Loc_unknown;
-        exn_continuation = None; },
-      I.Switch(is_scrutinee_int, isint_switch))
+      let isint_switch =
+        I.Let ( is_scrutinee_int,
+          Not_user_visible,
+          Pintval,
+          Prim { prim = Pflambda_isint;
+            args = [Var scrutinee];
+            loc = Loc_unknown;
+            exn_continuation = None; },
+          I.Switch(is_scrutinee_int, isint_switch))
       in
       isint_switch,
       ((const_cont, const_switch)::(block_cont, block_switch)::wrappers)
   in
   cps_non_tail scrutinee (fun scrutinee ->
-      let switch, wrappers = build_switch scrutinee wrappers in
-      List.fold_left (fun body (cont, action) ->
-          I.Let_cont {
-            name = cont;
-            is_exn_handler = false;
-            params = [];
-            recursive = Nonrecursive;
-            body;
-            handler = action;
-          })
-        switch wrappers)
+    let switch, wrappers = build_switch scrutinee wrappers in
+    List.fold_left (fun body (cont, action) ->
+      I.Let_cont {
+        name = cont;
+        is_exn_handler = false;
+        params = [];
+        recursive = Nonrecursive;
+        body;
+        handler = action;
+      })
+      switch wrappers)
```

Comment:
For this particular piece of code, the previous version is better; I've opened a new PR (#431) that should make `ocp-indent` pick the correct indentation. Once it's merged, you should be able to get the correct indentation back automatically.